Apps/phvisualping/visualping.json

```diff
@@ -0,0 +1,536 @@
+{
+    "appid": "394f9efc-5ac5-4ab4-8e0e-fdc59a2a5e0a",
+    "name": "visualping",
+    "description": "The visualping app is a convenient tool to monitor websites changes",
+    "type": "siem",
+    "product_vendor": "visualping",
+    "logo": "logo_visualping.svg",
+    "logo_dark": "logo_visualping_dark.svg",
+    "product_name": "visualping",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "contributors": "Matt Sayar",
```

Comment:
Did @mattsayar-splunk write and/or upload this?

Reply:
Yes